tools/envoy_headersplit/headersplit_test.py

```diff
@@ -0,0 +1,107 @@
+# Lint as: python3
+"""Tests for headersplit."""
+
+import unittest
+import headersplit
+import io
+import clang.cindex
+from clang.cindex import TranslationUnit, Index, CursorKind
+clang.cindex.Config.set_library_path("/opt/llvm/lib")
+
+
+class HeadersplitTest(unittest.TestCase):
+  # A header contains a simple class print hello world
```

Comment:
These class-level constants appear unused; can they be deleted?

Reply:
they are removed now